Microsoft.Toolkit.Uwp.Services/Services/OneDrive/OneDriveStorageFile.cs

```diff
@@ -85,6 +88,31 @@ public OneDriveStorageFile(IOneDriveClient oneDriveProvider, IItemRequestBuilder
         }
 
         /// <summary>
+        /// Creates a background download for the current file
+        /// </summary>
+        /// <param name="destinationFile">A <see cref="StorageFile"/> to which content will be downloaded</param>
+        /// <param name="completionGroup">The <see cref="BackgroundTransferCompletionGroup"/> to which should <see cref="BackgroundDownloader"/> reffer to</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request</param>
+        /// <returns>The created <see cref="DownloadOperation"/></returns>
+        public async Task<DownloadOperation> CreateBackgroundDownloadAsync(StorageFile destinationFile, BackgroundTransferCompletionGroup completionGroup = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (destinationFile == null)
+            {
+                throw new ArgumentNullException(nameof(destinationFile));
+            }
+
+            var requestMessage = Provider.Drive.Items[OneDriveItem.Id].Content.Request().GetHttpRequestMessage();
+            await Provider.AuthenticationProvider.AuthenticateRequestAsync(requestMessage).AsAsyncAction().AsTask(cancellationToken);
+            var downloader = completionGroup == null ? new BackgroundDownloader() : new BackgroundDownloader(completionGroup);
+            foreach (var item in requestMessage.Headers)
+            {
+                downloader.SetRequestHeader(item.Key, item.Value.First());
+            }
+
+            return await Task.Run(() => downloader.CreateDownload(requestMessage.RequestUri, destinationFile), cancellationToken);
```

Comment:
@ScottIsAFool [The MSDN](https://docs.microsoft.com/en-us/uwp/api/windows.networking.backgroundtransfer.backgrounddownloader#Windows_Networking_BackgroundTransfer_BackgroundDownloader_CreateDownload_Windows_Foundation_Uri_Windows_Storage_IStorageFile_) says that if you create large number of requests then performance of UI can get degraded and we don't know if users won't use this method in a loop.

> Creating a large number of transfers on the main UI thread with CreateDownload can result in degraded performance of your app's UI. If you are queuing up a large number of transfers, it is recommended that you call CreateDownload on a background worker thread as in the following example.